packages/node-fhir-server-core/package.json

```diff
@@ -37,8 +37,8 @@
     "verbose": true
   },
   "dependencies": {
-    "@asymmetrik/fhir-response-util": "^1.2.4",
-    "@asymmetrik/sof-scope-checker": "^1.0.7",
+    "@projecttacoma/fhir-response-util": "^1.2.4",
+    "@projecttacoma/sof-scope-checker": "^1.0.7",
```

Comment:
Now that we are only publishing `@projecttacoma/node-fhir-server-core` for now, we can revert these dependencies back to using asymmetrik